test/gtest/uct/test_rc_flow_control.cc

```diff
@@ -0,0 +1,76 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2016.  ALL RIGHTS RESERVED.
+* Copyright (C) UT-Battelle, LLC. 2016. ALL RIGHTS RESERVED.
+* Copyright (C) ARM Ltd. 2016.All rights reserved.
+* See file LICENSE for terms.
+*/
+
+extern "C" {
+#include <uct/api/uct.h>
+#include <uct/ib/rc/base/rc_ep.h>
+#include <uct/ib/rc/base/rc_iface.h>
+}
+#include <common/test.h>
+#include "uct_test.h"
+
+class test_rc_flow_control : public uct_test {
+public:
+    void initialize() {
```

Comment:
you can just override "voirtual void init()" here and would not to explicitly call initialize() from the test body